src/Nethermind/Nethermind.WebSockets/ISocketHandler.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Nethermind.WebSockets
+{
+    public interface ISocketHandler : IDisposable
```

Comment:
Can you add documentation comment explaining what it is (especially one can confuse ISocketHandler with ISocketClient)